hcontext/requestid.go

```diff
@@ -26,14 +26,16 @@ var headersToSearch = []string{
 // If one is found, it appends a new request ID and sets the comma separated value as the header.
 // If one is not found, it sets a new request ID as the header.
 func FromRequest(r *http.Request) (id string, ok bool) {
+	newRID := uuid.New().String()
+
 	for _, try := range headersToSearch {
 		if id = r.Header.Get(try); id != "" {
-			newRequestID := uuid.New().String() + "," + id
-			return newRequestID, true
+			newRID = newRID + "," + id
```

Comment:
@jboursiquot-sfdc is this valid go?

Reply:
Yes, though it would be more idiomatic to do something like this: `newRID = fmt.Sprintf("%s,%s", newRID, try)`